playbook-website/app/javascript/site_styles/_visual_guidelines.scss

```diff
@@ -1,25 +1,30 @@
 @import "playbook-ui/dist/tokens/spacing";
 @import "playbook-ui/dist/tokens/screen_sizes";
+@import "playbook-ui/dist/tokens/opacity";
 
 .visual_guidelines {
   .header-text {
     max-width: 600px;
     margin: auto;
   }
+
   &_hero {
     padding: $space_xl * 1.75 !important;
   }
+
   .token-wrapper {
     max-width: 1242px;
     margin: auto;
   }
-  @include break_at(breakpoint("sm")){
+
+  @include break_at(breakpoint("sm")) {
     .visual_guidelines_hero {
       padding: $space_xl !important;
     }
   }
+
   .positioning-guideline {
-    & > * {
+    &>* {
```

Comment:
🤔

Reply:
That isn't me? I think Dat is doing positioning.

Reply:
I am not sure what I am supposed to do here. This was here before I made changes to the file.